docs/source/tutorials.rst

```diff
@@ -105,43 +107,273 @@ After installing the AS2FM packages as described in the :ref:`installation secti
 The output is a JANI file called `main.jani` that will be located in the same folder.
 
 
-.. _jani_conversion:
 
-How to convert from CONVINCE robotic JANI to plain JANI?
------------------------------------------------------------
 
-.. note::
-    The CONVINCE robotic JANI extension is not related to the modeling of robotics system using ROS, unlike the SCXML approach. The following tutorial targets users that want to develop a JANI model of a robotics system by hand.
 
-We provide a Python script to convert models describing the system and its environment together, given in the CONVINCE robotics JANI flavor as specified in the `data model repository <https://github.com/convince-project/data-model>`_, into `plain JANI <https://jani-spec.org>`_ accepted as input by model checkers.
 
+.. _full_tutorial:
 
-Running the Script
-```````````````````
+Hands-on in-depth tutorial including verification: Fetch & Carry
+-----------------------------------------------------------------
+
+In this tutorial you will learn within around one hour how a fetch and carry robot scenario can be modeled in SCXML and how linear temporal logic properties can be verified on it. You translate the model of the robot and its environment with AS2FM into JANI for verification with `SMC Storm <https://github.com/convince-project/smc_storm>`_, our statistical model checking tool.
+You will observe fulfilled and violated properties. By updating the model with more involved functionality in terms of probabilistic behavior and additional features in the behavior tree, you make in the end sure that the required properties hold.
+
+We assume some background in computer science or as a robotics developer but no knowledge about formal methods and model checking is required.
+
+
+Reference Model: Fetch & Carry Robot
+`````````````````````````````````````
+
+For this tutorial we use the model defined here: `tutorial_fetch_and_carry <https://github.com/convince-project/AS2FM/tree/main/test/jani_generator/_test_data/tutorial_fetch_and_carry>`_.
```

Comment:
I would suggest, the participants first copy the model somewhere. Or we put it in a special tutorial or example folder. Having them work in our test data feels like the wrong place.